Add docs about patch run (#332)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/ci-cd/github-workflow/github-workflow.md

- Create/Update catalog entities - invoked with the `UPSERT` operation, receives the identifier and other properties of a new entity or an entity that needs to be updated;
- Get catalog entities - invoked with the `GET` operation, receives the identifier of an existing entity and retrieves it for use in your CI.

> - Update Running action - invoked with the `PATCH_RUN` operation, receives the identifier and other properties of a running action or an action that needs to be updated.

Reviewer: ```suggestion
- Update a running action - invoked with the `PATCH_RUN` operation, receives the identifier of an existing action run along with other properties of the run that need to be updated.
```
